README.md

```diff
@@ -5,3 +5,8 @@ The AirPort Air Quality (APAQ) Predictor is an API tool that can be used to pred
 1. Create a virtual environment by executing `make` in the root.
 2. Activate the virtual environment by executing `. venv/Scripts/activate` (for Windows) or `. venv/bin/activate` (for Mac/Linux) also in the root directory.
 3. Run the program by executing `make run` in the root directory.
+
+## Train a Random Forest Model
+1. Execute: `python -m src.model.main --aug-airports-load-file DATA_FILE_PATH` where DATA_FILE_PATH is the local path to the training dataset 
```

Comment:
I don't really like the name `--aug-airports-load-file` - could we change this to something a bit more sensible?